pro/generate-documentation-qmd/action.yml

```diff
@@ -45,11 +45,17 @@ inputs:
       Release of Viash tools to use.
     required: false
     default: 'latest'
+  working_directory:
```

Comment:
Can you better describe what the use of this parameter is, and perhaps also include it in the `pro/generate-documentation-qmd/README.md`?

Reply:
Added!